setup.cfg

```diff
@@ -2,7 +2,7 @@
 verbosity=1
 detailed-errors=1
 with-coverage=1
-cover-package=mslice.presenters,mslice.plotting.globalfiguremanager
+cover-package=mslice.presenters,mslice.plotting.globalfiguremanager,mslice.util.mantid,mslice.scripting,mslice.workspace 
```

Comment:
It looks slightly lengthy line (or maybe it's just my eyes). If possible, consider splitting into two lines?